### PR TITLE
Add information about ember-collection

### PR DESCRIPTION
**Pull request by raytiley**

### CODE REVIEW

**Comment by tim-evans** [comment]
This looks good to me. Let's get this merged in ASAP.

File: README.md [hunk 1]
```diff
@@ -37,6 +40,10 @@ Please, take a look at our live [demo](http://emberjs.com/list-view) and [jsbin]
 Please, attach code samples or links to [jsbin](http://emberjs.jsbin.com/) or [jsfiddle](http://jsfiddle.net/).
 It would help us greatly to help you and to improve ember list view.
 
+## Compatibility
+
+*ListView* is compaitable with wember versions `<= 1.12.x`. It is important to note that *ListView* is not compaitable with the Glimmer rendering engine. [Ember Collection](https://github.com/emberjs/ember-collection) is ment to replace *ListView* as the ember solution to rendering large lists. If you need this functionality for your `>= 1.13.x` ember app, please checkout [Ember Collection](https://github.com/emberjs/ember-collection).
```
Comment (Robdel12) [inline README.md]: "wember" :P

Comment (tim-evans) [inline README.md]: Nice catch :P

Comment (raytiley) [inline README.md]: ugh... I hate writing docs... but someone has to do it. Will push the fix soon.

**Comment by stefanpenner** [comment]
LGTM

**Comment by raytiley** [comment]
> LGTM – when the above comment has been resolved.

Ok fixed the type. Hit the button.

**Pull request merged**